Makefile

```
@@ -22,32 +26,32 @@ compile:
 	@echo "==> Building the project"
 	@env CGO_ENABLED=0 GOCACHE=/tmp/go-cache GOOS=linux GOARCH=amd64 go build -a -o cmd/oras-csi-plugin/${NAME} cmd/oras-csi-plugin/main.go
 
-push-dev: build-dev
+push-dev: build-dev ## Build and push images to devtag
 	@echo "==> Publishing DEV $(DOCKER_REGISTRY)/oras-csi-plugin:$(DEVTAG)"
 	@docker push $(DOCKER_REGISTRY)/oras-csi-plugin:$(DEVTAG)
 	@docker push $(DOCKER_REGISTRY)/oras-csi-plugin:latest
 	@echo "==> Your DEV image is now available at $(DOCKER_REGISTRY)/oras-csi-plugin:$(DEVTAG)"
 
-push: build
+push: build ## Build and push images for DRIVER_VERSION
 	@echo "==> Publishing $(DOCKER_REGISTRY)/oras-csi-plugin:$(DRIVER_VERSION)"
 	@docker push $(DOCKER_REGISTRY)/oras-csi-plugin:$(DRIVER_VERSION)
 	@docker push $(DOCKER_REGISTRY)/oras-csi-plugin:latest
 	@echo "==> Your published image is now available at $(DOCKER_REGISTRY)/oras-csi-plugin:$(DRIVER_VERSION)"
 
-build: compile
+build: compile ## Build images for DRIVER_VERSION
 	@echo "==> Building docker images"
 	@docker build -t $(DOCKER_REGISTRY)/oras-csi-plugin:$(DRIVER_VERSION) cmd/oras-csi-plugin
 	@docker build -t $(DOCKER_REGISTRY)/oras-csi-plugin:latest cmd/oras-csi-plugin
 
-build-dev: compile
+build-dev: compile ## Build images for DEVTAG
 	@echo "==> Building DEV docker images"
 	@docker build -t $(DOCKER_REGISTRY)/oras-csi-plugin:$(DEVTAG) cmd/oras-csi-plugin
 	@docker build -t $(DOCKER_REGISTRY)/oras-csi-plugin:latest cmd/oras-csi-plugin
 
-clean:
+clean: ## Deletes driver and all pods
```

```suggestion
clean: ## Deletes driver
```
Thankfully I removed the "and all pods" part :)